pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -95,3 +101,10 @@ func ObserveStorageURLs(genericListers configobserver.Listers, recorder events.R
 
 	return
 }
+
+func isValidIP(ip string) bool {
```

Comment:
this does really need a helper? It wraps one func in another with zero added value or abstraction.

Reply:
valid point, I was not sure if that was the only validity check we wanted. Thought we would add more checks in future hence premature-abstraction if the is even a term :)

I will drop it for time being, thanks